pkg/clients/aws/client.go

```diff
@@ -127,7 +160,7 @@ func (c *client) CheckoutRancherLicense(ctx context.Context, l types.GrantedLice
 	res, err := c.lm.CheckoutLicense(ctx, &lm.CheckoutLicenseInput{
```

Comment:
This is for line 122, but GitHub won't let me review that line. Do we know that `l.LicenseArn` is not nil?

Reply:
I can put a stopgap in place. That being said, I would be very surprised to see a missing arn here.